src/tuf/index.ts

```diff
@@ -32,31 +32,43 @@ export interface TUFOptions {
   rootPath?: string;
 }
 
+export interface TUF {
+  getTarget(targetName: string): Promise<string>;
```

Comment:
Is this not also exported?
```suggestion
  getTarget(targetName: string): Promise<string>;
  refresh(): Promise<void>;
```

Reply:
I'm specifically not exporting the refresh since I'm already calling it before I return the client. This will prevent someone from accidentally invoking it again (and incurring the reload overhead)

Reply:
Ah cool but looks like it's public in the below class?